deephub/models/feeders/tfrecords/__init__.py

```diff
@@ -246,6 +248,14 @@ def get_tf_dataset(self, epochs: Optional[int] = None) -> tf.data.Dataset:
         # buffer_size=None is autotuning the appropriate prefetching buffer size
         return batches.prefetch(buffer_size=self.prefetch)
 
```

Comment:
The extension for the files should be as an input from the config as we discussed. I don't like the fact that we have to scan the 1st file in order to find the extension type. Also, (I am not 100% sure), but if not all files are with the same extension maybe by default the tensor flow API throws an extension. You can check it.